packages/experimental/Switch/src/Switch.styling.ts

```diff
@@ -103,6 +105,7 @@ export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, Sw
       (tokens: SwitchTokens, theme: Theme) => ({
         style: {
           color: tokens.color,
+          ...(isMobile && { flexShrink: 1 }),
```

Comment:
I am not sure if we should do this as part of Switch component. What would be the behaviour for a text which say spans 4 lines - do we expands the component to take vertical space to render all the text? Or truncate after a specific limit. 
Can user not instead customise/style the properties for the label if it needs to be too long?